daemon/src/main/java/org/lsposed/lspd/cli/Main.java

```diff
@@ -0,0 +1,142 @@
+package org.lsposed.lspd.cli;
+
+import android.os.Binder;
+import android.os.IBinder;
+import android.os.Parcel;
+import android.os.RemoteException;
+import android.os.ServiceManager;
+
+import com.beust.jcommander.ParameterException;
+
+import org.lsposed.lspd.ILSPManagerService;
+import org.lsposed.lspd.cli.exception.FormatException;
+import org.lsposed.lspd.cli.exception.NoEnoughArgumentException;
+import org.lsposed.lspd.cli.exception.NotXposedModuleException;
+import org.lsposed.lspd.cli.exception.RunException;
+import org.lsposed.lspd.cli.exception.UninstalledPackageException;
+import org.lsposed.lspd.cli.exception.UnknownCommandException;
+import org.lsposed.lspd.cli.handler.ICommandHandler;
+import org.lsposed.lspd.cli.handler.LogHandler;
+import org.lsposed.lspd.cli.handler.ModulesHandler;
+import org.lsposed.lspd.cli.handler.ScopeHandler;
+import org.lsposed.lspd.cli.handler.StatusHandler;
+import org.lsposed.lspd.service.ILSPApplicationService;
+import org.lsposed.lspd.util.SignInfo;
+
+import static org.lsposed.lspd.cli.Utils.CMDNAME;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class Main {
+
+    private static final Map<String, ICommandHandler> handlerMap = new HashMap<>();
```

Comment:
use JCommander to handle subcommand instead of querying a map

Reply:
@yujincheng08 thank you for your suggestion, now looks a bit better.
I have some doubts, on methods run now return an integer, but it's always 0 because when there is an error there is exception.